### DISCUSSION TITLE
Interactive (yml) issue templates not supported

### DISCUSSION THREAD

**Discussion by htcfreek** [community/community#3975] [Mobile]
The mobile app didn't support ui based issue templates (yml templates).

This is a problem because some repositories have switched their templates.
Example:
- Microsoft/WindowsTerminal
- Microsoft/PowerToys

Because of this fact these repos didn't support creating issues on mobile app anymore.

**Comment by elisealix22** [comment]
Thank you again for your feedback on the app! We are tracking adding Issue Forms support to our apps internally and I added your note to the issue. I'm sorry for the inconvenience in the meantime. If you have more bugs specific to the GitHub for Mobile apps, don't hesitate to reach back out!

**Reply by htcfreek** [reply]
Any updates on this? It is really annoying to always need to open the GH webpage.
